terraform/workspace-variables/production_app_env.yml

```diff
@@ -0,0 +1,10 @@
+---
+RACK_ENV: production
+RAILS_ENV: production
+NODE_ENV: production
+RAILS_LOG_TO_STDOUT: true
+RAILS_MAX_THREADS: 5
+RAILS_SERVE_STATIC_FILES: true
+SERVICE_TYPE: web
+DATABASE_URL: postgres://postgres@localhost:5432
+REGISTER_AND_PARTNER_URL: "https://manage-training-for-early-career-teachers.education.gov.uk/"
```

Comment:
Does this mean we're going to get polled with a bad API token? 😂 That's fine, I'll just know it ignore it

Reply:
Yeah, I expect there will be a 401 or 403 every 5 mins :P Sorry about that, want to test it out in staging first, but I don't want our prod to have your staging ECTs and mentors :D